testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -1446,17 +1458,30 @@ fn multiple_miners() {
     );
 
     let mut run_loop_2 = boot_nakamoto::BootRunLoop::new(conf_node_2.clone()).unwrap();
+    let run_loop_stopper_2 = run_loop_2.get_termination_switch();
     let rl2_coord_channels = run_loop_2.coordinator_channels();
     let Counters {
         naka_submitted_commits: rl2_commits,
         ..
     } = run_loop_2.counters();
-    let _run_loop_2_thread = thread::Builder::new()
+    let run_loop_2_thread = thread::Builder::new()
         .name("run_loop_2".into())
         .spawn(move || run_loop_2.start(None, 0))
         .unwrap();
 
     signer_test.boot_to_epoch_3();
+
+    wait_for(120, || {
```

Comment:
This was a precaution to ensure that both nodes have caught up to each other before proceeding. Not 100% sure it was necessary, but is safe to do and I think will prevent unintended race conditions.